### PR TITLE
enable Jakarta for transaction context

### PR DESCRIPTION
**Pull request by njr-11**
Update the transaction context autofeature to support Jakarta.
Enable the main test bucket for EE Concurrency once transaction context is working in Jakarta.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent_fat/test-bundles/test.resource/src/test/resource/internal/TestResource.java [hunk 1]
```diff
@@ -68,12 +66,18 @@ public Object run() throws Exception {
...
+                    ServiceReference<?> mgdExecSvcRef = bundleContext.getServiceReference("jakarta.enterprise.concurrent.ManagedExecutorService");
+                    if (mgdExecSvcRef == null)
+                        mgdExecSvcRef = bundleContext.getServiceReference("javax.enterprise.concurrent.ManagedExecutorService");
+
                     String displayId = (String) mgdExecSvcRef.getProperty("config.displayId");
                     if (!"managedExecutorService[DefaultManagedExecutorService]".equals(displayId))
                         throw new Exception("Unexpected ManagedExecutorService with highest service.ranking: " + displayId);
 
-                    ServiceReference<ManagedScheduledExecutorService> mgdSchedExecSvcRef = bundleContext.getServiceReference(ManagedScheduledExecutorService.class);
+                    ServiceReference<?> mgdSchedExecSvcRef = bundleContext.getServiceReference("java.enterprise.concurrent.ManagedScheduledExecutorService");
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent_fat/test-bundles/test.resource/src/test/resource/internal/TestResource.java:77]: Missing "x" in "javax".  Should be:
javax.enterprise.concurrent.ManagedScheduledExecutorService

Comment (njr-11) [reply dev/com.ibm.ws.concurrent_fat/test-bundles/test.resource/src/test/resource/internal/TestResource.java]: Good catch - added a commit to fix it.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#run-libby-bot

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Pull request merged**